docs_user/modules/proc_using-new-subnet-ranges.adoc

```diff
@@ -8,6 +8,9 @@ You must size the new subnet appropriately to accommodate the new control
 plane services. There are no specific requirements for the
 existing deployment allocation pools that are already consumed by the {OpenStackShort} environment.
 
+[IMPORTANT]
+Defining a new subnet for Storage it is not supported.
```

Comment:
What's the rationale behind this?

Reply:
You mean why we can't modify storage network during adoption? If that's the case, the reason it's related to nova and cinder IIRC, nova doesn't support changing the network where it stores the VM's volumes. (I'm not an expert neither on nova nor cinder, will try to get a better explanation on nova/cinder team to proper include an explanation)